build/vega-lite-schema.json

```diff
@@ -798,7 +798,7 @@
         "labelPadding": {
```

Comment:
Can we remove the schema updates from this commit so we can revert it easily?

Reply:
It won't help anyway because we change import in many places later. 

The one that should be reverted is "fix: casting"